### PR TITLE
Story/tp 188 group admin user administration

### PR DESCRIPTION
**Pull request by jussiviitamaki**
**Actions necessary for applying the changes:** *(for example composer install; drush updb; drush cim)*

**Testing instructions:**
- [x] Login
- [x] Go to group entity
- [x] Add existing or new subgroup
- [x] Confirm current user has been added as admin to subgroup

### CODE REVIEW

**Review by ZeiP — CHANGES_REQUESTED** [review]
CI on epäonnistunut: ”- The lock file is not up to date with the latest changes in composer.json, it is recommended that you run `composer update` or `composer update <package name>`.”

File: composer.json [hunk 1]
```diff
@@ -83,6 +85,7 @@
         "drupal/search_api": "^1.23",
         "drupal/search_api_autocomplete": "^1.6",
         "drupal/smart_date": "^3.5",
+        "drupal/subgroup": "^1.0",
```
Comment (ZeiP) [inline composer.json:88]: Is this needed? Doesn't seem to be enabled.

File: config/sync/core.extension.yml [hunk 2]
```diff
@@ -43,6 +43,7 @@ module:
   flag: 0
   flag_lists: 0
   gcontent_moderation: 0
+  ggroup: 0
```
Comment (ZeiP) [inline config/sync/core.extension.yml:46]: Ggroup is only used a few hundred times, is not under security upgrades and the devs don't seem to be too active. Did you evaluate the code and the support situation?

File: public/modules/custom/hel_tpm_search_autosuggest/js/hel-tpm-search-autosuggest.js [hunk 3]
```diff
@@ -140,6 +140,9 @@
             Drupal.behaviors.hel_tpm_search_autocomplete.showHideAutocomplete(this, context);
           });
 
+        $(form).closest('form').on('submit', function(e) {
```
Comment (ZeiP) [inline public/modules/custom/hel_tpm_search_autosuggest/js/hel-tpm-search-autosuggest.js:143]: What are these changes? Could you add them to the testing instructions?

**Review by ZeiP — CHANGES_REQUESTED** [review]
Is there a way to test the other subgroup functionality that apparently should come with the PR?

File: composer.json (hunk 1 above)
Comment (jussiviitamaki) [reply composer.json]: Removed subgroup module from composer

File: config/sync/core.extension.yml (hunk 2 above)
Comment (jussiviitamaki) [reply config/sync/core.extension.yml:46]: Did a quick code review previously and code seems to be within best practices and well done. There's quite a lot to be done feature wise but that would be the case either way.

Compared to subgroup-module the usage statistics (358 vs 361) are essentially the same. 
So is the maintenance status (subgroup last update may 2021 and ggroup february 2021).

Basically the only difference between the two when it comes to maintenance is that subgroup has stable release and ggroup does not.

File: public/modules/custom/hel_tpm_search_autosuggest/js/hel-tpm-search-autosuggest.js (hunk 3 above)
Comment (jussiviitamaki) [reply public/modules/custom/hel_tpm_search_autosuggest/js/hel-tpm-search-autosuggest.js]: Whoops wrong branch.. Moved these changes to another PR

**Pull request merged**